Update custom-branch-settings.md (#4266)
fishtown-analytics/docs.getdbt.com: website/docs/faqs/Environments/custom-branch-settings.md

---
title: How do I use the 'Custom Branch' settings in a dbt Cloud Environment?
description: "Use custom code from your repository"

> sidebar_label: 'Custom Branch settings'

Author: ```suggestion
sidebar_label: 'Custom branch settings'
```

## Development

> In a development environment, the default branch (usually named `main`) is a read-only branch in your connected repositories, which allows you to create new branches for development based on it.  

Reviewer: ```suggestion
In a development environment, the default branch (usually named `main`) is a read-only branch in your connected repositories, which allows you to create new branches for development from it.  
```

> Specifying a **Custom branch** overrides the default behavior. It makes the custom branch 'read-only' and enables you to create new development branches based on it. This also means you can't edit this custom branch directly.

Reviewer: ```suggestion
Specifying a **Custom branch** overrides the default behavior. It makes the custom branch 'read-only' and enables you to create new development branches from it. This also means you can't edit this custom branch directly.
```
